server/plugin.go

```diff
@@ -41,6 +41,19 @@ const (
 
 	metricsExposePort          = ":9094"
 	updateMetricsTaskFrequency = 15 * time.Minute
+
+	discardedReasonUnableToGetMMData         = "unable_to_get_mm_data"
+	discardedReasonUnableToUploadFileOnTeams = "unable_to_upload_file_on_teams"
+
+	actionSourceMattermost = "mattermost"
+	directMessageTrue      = "true"
```

Comment:
Maybe this as well ?

Reply:
Yes, there are some constants being repeated because we currently don't have a common constant package. Added a TODO comment regarding the same

Reply:
Cant we just export this variable if this does not cause import cycle. @ayusht2810

Reply:
removed repeated variables